pom.xml

```diff
@@ -41,6 +41,8 @@
           <compilerArgs>
             --enable-preview
           </compilerArgs>
+          <source>19</source>
+          <target>19</target>
```

Comment:
Do we need these two lines when we already have configured the version in properties? 
 <properties>
    <maven.compiler.source>19</maven.compiler.source>
    <maven.compiler.target>19</maven.compiler.target>
    
   When having configurations in several places it's harder to maintain the code.

Reply:
Thank you for reviewing my code. I'm going to delete these two lines.